wgpu-types/src/lib.rs

```diff
@@ -1739,13 +1775,24 @@ pub enum AstcChannel {
     Hdr,
 }
 
-/// Underlying texture data format.
+/// Format in which a texture’s texels are stored in GPU memory.
+///
+/// Certain formats additionally specify a conversion.
+/// When these formats are used in a shader, the conversion automatically takes place when loading
+/// from or storing to the texture.
 ///
-/// If there is a conversion in the format (such as srgb -> linear), the conversion listed here is for
-/// loading from texture in a shader. When writing to the texture, the opposite conversion takes place.
+/// * `Unorm` formats linearly scale the integer range of the storage format to a floating-point
+///   range of 0 to 1, inclusive.
+/// * `Snorm` formats linearly scale the integer range of the storage format to a floating-point
+///   range of &minus;1 to 1, inclusive.
```

Comment:
Is it worth mentioning the Snorm edge case here (Rgba8Snorm for example has both -127 and -128 both map to `-1.0`)?

Reply:
Yeah it applies to all Snorm types (https://docs.vulkan.org/spec/latest/chapters/fundamentals.html#fundamentals-fixedfpconv).

Oof yeah that's just a typo. I think changing to [-127, 127] etc is reasonable.

Reply:
Done.

Reply:
Nice!